apis/bigtable/v1alpha1/authorizedview_types.go

```diff
@@ -33,7 +33,6 @@ type BigtableAuthorizedViewSpec struct {
 	// The BigtableAuthorizedView name. If not given, the metadata.name will be used.
 	ResourceID *string `json:"resourceID,omitempty"`
 
-	// +required
 	BigtableAuthorizedViewParent `json:",inline"`
```

Comment:
oh this is an interesting corner case. It may be kubebuilder tag specific but that `required: - ""` below is odd! Thanks for removing it!